### PR TITLE
Fix Sunrise City Server

### PR DESCRIPTION
**Pull request by skepiaa**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [x] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [x] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [x] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [x] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [x] `regions`: a list of regions where you have servers located that service your players *(ex. `["NA", "EU", "AS"]`)*
  - [x] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is `512` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [x] My background is a `png` file.
* [x] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [x] My background has an aspect ratio of 16:9.
* [x] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [x] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.

### CODE REVIEW

**Comment by JustSkeppYT** [comment]
This commit is to update the old server details (sunrisecity) to add in a port.

**Review by itsJordan — CHANGES_REQUESTED** [review]
There are now new requirements for ServerMappings updates/submissions.

Please review the README and include all missing information.

**Comment by JustSkeppYT** [comment]
Updated as per following the README.

**Review by itsJordan — CHANGES_REQUESTED** [review]
You are still missing the following:

- primaryLanguage
- languages
- primaryRegion
- regions
- gameTypes

**Comment by JustSkeppYT** [comment]
> * gameTypes

How does gameTypes work, RPG?

**Review by itsJordan — CHANGES_REQUESTED** [review]
I cannot approve this PR as Lunar Client does not support ServerMappings with specified ports, and we cannot map this IP Address to a server as the server is not consistent.

Could you either:
1. Obtain a dedicated IP Address that your domain can be pointed at.
2. Configure an SRV record within your DNS configuration that points to your specific port:

Example:
- type: SRV Record

- value: _minecraft
- protocol: _tcp.play
- priority: 0
- weight: 5
- port: 2260
- target: play.sunrisecity.uk
- ttl: default/auto

**Comment by JustSkeppYT** [comment]
Hmm, everything on that list is correct to our SRV already, not really sure why it doesn't work.
image

**Comment by JustSkeppYT** [comment]
Does lunar client automatically forward the person into port 25565? That may be an issue.

**Comment by itsJordan** [comment]
> Does lunar client automatically forward the person into port 25565? That may be an issue.

Lunar will direct the player to 25565 by default, but will direct connections to an alternate port if an SRV record exists just like vanilla.

Your SRV record is resolving fine for me (see img) and I can connect to the server using 'play.sunrisecity.uk' 
image

Are you sure players are not consistently being connected to the correct place?

**Comment by JustSkeppYT** [comment]
> Are you sure players are not consistently being connected to the correct place?

I'm not really sure what that means

**Comment by itsJordan** [comment]
> > Are you sure players are not consistently being connected to the correct place?
> 
> I'm not really sure what that means

When I originally queried your reason for using a specific port, you responded with:

"
Yeah and anyways fort he Port 2260, the server sometimes redirect them to another server if we don't use port 2260.
Since due to the fact that our server doesn;t have dedicated IP
In which in some circumstances, the players may be connected to another server.
"

We've taken that to mean "Unless the player is directly connected to 'play.sunrisecity.uk:2260', they will not always be connected to the correct server". Is that incorrect?

**Comment by JustSkeppYT** [comment]
> We've taken that to mean "Unless the player is directly connected to 'play.sunrisecity.uk:2260', they will not always be connected to the correct server". Is that incorrect?

That is correct.

**Comment by JustSkeppYT** [comment]
Although it should be fixed after contacting the host and the chance of being connected to that server is extremely low to none.

**Comment by JustSkeppYT** [comment]
I tried to connect with port :25565 and it worked, not really sure what's wrogn with that.

**Comment by JustSkeppYT** [comment]
@itsJordan Any updates?

**Comment by itsJordan** [comment]
> @itsJordan Any updates?

Can you confirm that all connections to 'play.sunrisecity.uk' will route players to the correct server?

**Comment by JustSkeppYT** [comment]
> Can you confirm that all connections to 'play.sunrisecity.uk' will route players to the correct server?

According to the person who owns/setups the domain+cloudflare, for the past months it has never redirected us to any other server and works perfectly fine redirecting us to the correct IP. 'play.sunrisecity.uk' has been used for the past months without anything happening.

File: servers/sunrisecity/metadata.json [hunk 1]
```diff
@@ -4,10 +4,29 @@
     "addresses": [
         "sunrisecity.uk"
     ],
-    "primaryAddress": "play.sunrisecity.uk",
+    "primaryAddress": "play.sunrisecity.uk:2260",
```
Comment (itsJordan) [inline servers/sunrisecity/metadata.json:7]:
```suggestion
    "primaryAddress": "play.sunrisecity.uk",
```

**Review by itsJordan — APPROVED** [review]

**Pull request merged**